reference_data/platforms.yaml

```diff
@@ -61,7 +61,7 @@
       environment:
         DFLTCC: '0'
       use_bootstrap_container: false
-      macros:
+      secure_boot_macros:
```

Comment:
We need the same changes in flavors if they have secure boot macroses

Reply:
Now we don't have secure boot macroses in flavors